index.html

```diff
@@ -162,7 +162,7 @@ <h2>Graphic Design</h2>
     <!-- end of OfferedService -->
 
     <!-- NumbersCounter -->
-    <section class="container bg-gradient">
+    <section class="container bg-primary-color-gradient">
```

Comment:
bg-primary-color-gradient klases nera, yra bg-gradient :) čia reiktu pataisyti. toliau kur naudoja taip pat

Reply:
thanks to me :D